### ISSUE TITLE
Not working in incognito mode

### ISSUE THREAD

**Issue by GordianZ** [sparanoid/omnibox-ncr#1]
lol

**Comment by sparanoid** [comment]
Yeah it's not working the first time you request to google.com in incognito mode, a workaround is to first access www.google.com/ncr to gain the ncr cookie then it should work, feel free to pr for this issue :P

**Comment by ruchid123123** [comment]
Error de baixada: Invalid manifest
